Nova/src/main/kotlin/xyz/xenondevs/nova/tileentity/impl/agriculture/AutoFisher.kt

```diff
@@ -92,6 +91,9 @@ class AutoFisher(
             
             if (timePassed >= maxIdleTime) {
                 timePassed = 0
+
+                if (!DROP_EXCESS_ON_GROUND && inventory.isFull()) return
```

Comment:
You added this ``if`` at different locations for each tile-entity. 
I'd say move it in front of  `` energyHolder.energy -= energyHolder.energyConsumption`` in all machines. A machine that doesn't do anything shouldn't consume energy.